src/main/java/com/synopsys/integration/alert/provider/blackduck/descriptor/BlackDuckDistributionUIConfig.java

```diff
@@ -53,12 +53,14 @@ public BlackDuckDistributionUIConfig(final BlackDuckProvider provider) {
 
     @Override
     public List<ConfigField> createProviderDistributionFields() {
-        final ConfigField filterByProject = CheckboxConfigField.createRequired(CommonDistributionConfiguration.KEY_FILTER_BY_PROJECT, "Filter by project");
-        final ConfigField projectNamePattern = TextInputConfigField.create(CommonDistributionConfiguration.KEY_PROJECT_NAME_PATTERN, "Project name pattern", this::validateProjectNamePattern);
+        final ConfigField filterByProject = CheckboxConfigField.createRequired(CommonDistributionConfiguration.KEY_FILTER_BY_PROJECT, "Filter by project",
+            "If true, all projects will be included. Any notifications matching the configured notification types will be processed.");
```

Comment:
I think this should say "If false, all projects will be included..."

Reply:
So, this is a bit complicated. The UI shows "Include all projects" for this field, which is the opposite of filtering by projects.  We have handled this is in the font end for several versions.  At some point we will need to sync up the front end and the back end so  they are consistent. I'm in favor of updating the backend since that would not require a major version change and it would have the least impact to customers. But I don't think we should do that in this PR